openapi/smartevents_mgmt_v1.yaml

```diff
@@ -1028,28 +1007,21 @@ components:
           description: "The URL of this resource, without the protocol"
           type: string
           example: example.com/resource
-          nullable: false
         submitted_at:
-          format: date-time
-          type: string
-          nullable: false
+          $ref: '#/components/schemas/ZonedDateTime'
```

Comment:
Is this done automatically by the new version of quarkus?
I think nothing is going to change but I was not expecting this difference (same for the `nullable: false `)

Reply:
Yes, it's due to the new version of the lib